docs/docs/features/reward-distribution.md

```diff
@@ -3,55 +3,45 @@ sidebar_position: 2
+Sending and distributing rewards from consumer chains to the provider chain is handled by the [Reward Distribution sub-protocol](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md#reward-distribution).
 
-## Note
-The ICS distribution system works by allowing consumer chains to send rewards to a module address on the provider called the `ConsumerRewardsPool`.
-There is a new transaction type called `RegisterConsumerRewardDenom`. This transaction allows consumer chains to register denoms to be used as consumer chain rewards on the provider.
-The cost to register a denom is configurable (`ConsumerRewardDenomRegistrationFee` chain param) and the full amount of this fee is transferred to the community pool of the provider chain. Only denoms registered through this transaction are then transferred from the `ConsumerRewardsPool` to the `FeePoolAddress`, to be distributed out to delegators and validators.
-
-### Instructions for adding a denom
-The transaction must be carried out on the provider chain. Please use the `ibc/*` denom trace format.
+Consumer chains have the option of sharing their block rewards (inflation tokens and fees) with the provider chain validators and delegators.
+In replicated security, block rewards are periodically sent from the consumer to the provider according to consumer chain parameters using an IBC transfer channel. 
+This channel is created during consumer chain initialization, unless it is provided via the `ConsumerAdditionProposal` when adding a new consumer chain. 
+For more details, see the [reward distribution parameters](../introduction/params.md#reward-distribution-parameters). 
 
 :::tip
-```
-# reward denoms must be registered on the provider chain (gaia in this example)
-gaiad tx provider register-consumer-reward-denom ibc/3C3D7B3BE4ECC85A0E5B52A3AEC3B7DFC2AA9CA47C37821E57020D6807043BE9 --from mykey
-```
+Providing an IBC transfer channel (see `DistributionTransmissionChannel`) enables a consumer chain to re-use one of the existing channels to the provider for consumer chain rewards distribution. This will preserve the `ibc denom` that may already be in use. 
+This is especially important for standalone chain transitioning to become consumer chains. 
+For more details, see the [changeover procedure](../consumer-development/changeover-procedure.md).
 :::
 
-## Parameters
-:::tip
-The following chain parameters dictate consumer chain distribution amount and frequency.
-They are set at consumer genesis and `blocks_per_distribution_transmission`, `consumer_redistribution_fraction`
-`transfer_timeout_period` must be provided in every `ConsumerChainAddition` proposal.
-:::
+Reward distribution on the provider is handled by the distribution module - validators and delegators receive a fraction of the consumer chain tokens as staking rewards.
```

Comment:
```suggestion
Reward distribution on the provider is handled by the distribution module.
```

The other half of the sentence was confusing for me, and it's already written above anyways.
If you want to keep it, I would write instead
"- validator and delegators receive a fraction of the block rewards generated by the consumer chain as staking rewards."